CSS/components/alert.css

```diff
@@ -0,0 +1,28 @@
+.avatar{
+    border-radius: 100%;
+    min-width: 5rem;
+    min-height: 5rem;
+    cursor: pointer;
+}
+.avatar-l{
+    min-width: 6rem;
+    min-height: 6rem;
+}
+.avatar-xl{
+    min-width: 7rem;
+    min-height: 7rem;
+}
+.avatar-s{
+    min-width: 3rem;
+    min-height: 3rem;
+}
```

Comment:
Class naming could have been better, I suppose its a small avatar, so .avatar-sm would sound more meaningful

Reply:
Thank you, Arpan, 
Made the changes in the class naming